examples/tera_template/src/main.rs

```diff
@@ -0,0 +1,74 @@
+//! A Hello World example application for working with Gotham.
+
+extern crate gotham;
+extern crate hyper;
+extern crate mime;
+#[macro_use]
+extern crate lazy_static;
+#[macro_use]
+extern crate tera;
+
+use hyper::{Response, StatusCode};
+
+use gotham::helpers::http::response::create_response;
+use gotham::state::State;
+use tera::{Context, Tera};
+
+///Assuming the Rust file is at the same level as the templates folder
```

Comment:
Nitpick, but a space after the `///` and don't start the second line with a leading `,`.